build.cmd

```diff
@@ -4,35 +4,32 @@ SETLOCAL
 
 SET CACHED_NUGET=%LocalAppData%\NuGet\NuGet.exe
 SET SOLUTION_PATH="%~dp0src\CodeFormatter.sln"
-SET MSBUILD14_TOOLS_PATH="%ProgramFiles(x86)%\MSBuild\14.0\bin\MSBuild.exe"
-SET MSBUILD12_TOOLS_PATH="%ProgramFiles(x86)%\MSBuild\12.0\bin\MSBuild.exe"
-SET BUILD_TOOLS_PATH=%MSBUILD14_TOOLS_PATH%
+SET VSWHERELOCATION="%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe"
+ 
+IF NOT EXIST %VSWHERELOCATION% (
+  goto :error
+)
+
+for /f "usebackq tokens=*" %%i in (`%VSWHERELOCATION% -latest -products * -requires Microsoft.Component.MSBuild -property installationPath`) do (
```

Comment:
Do you want `-prerelease` here?  That way dogfood builds work too.

Reply:
Sure, I'll do that.